### ISSUE TITLE
I only know the parameter /S, which is for silent mode installation. Does tesseract.exe have other parameters? I want to install it in a specific location.

### ISSUE THREAD

**Issue by qzd1989** [tesseract-ocr/tesseract#4360]
### Current Behavior

tesseract-ocr-w64-setup-5.5.0.20241111.exe /S

### Expected Behavior

tesseract-ocr-w64-setup-5.5.0.20241111.exe /S  /D="C:\tesseract"

such as /D

### Suggested Fix

sorry, I don't know the param of silent mode installation params.
I asked at here but no one to help me..
https://groups.google.com/g/tesseract-ocr/c/biZbztxEbv8

### tesseract -v

tesseract v5.5.0.20241111
 leptonica-1.85.0
  libgif 5.2.2 : libjpeg 8d (libjpeg-turbo 3.0.4) : libpng 1.6.44 : libtiff 4.7.0 : zlib 1.3.1 : libwebp 1.4.0 : libopenjp2 2.5.2
 Found SSE4.1
 Found libarchive 3.7.7 zlib/1.3.1 liblzma/5.6.3 bz2lib/1.0.8 liblz4/1.10.0 libzstd/1.5.6
 Found libcurl/8.11.0 Schannel zlib/1.3.1 brotli/1.1.0 zstd/1.5.6 libidn2/2.3.7 libpsl/0.21.5 libssh2/1.11.0

### Operating System

Windows 11

### Other Operating System

Windows 11 Professional  22H2

**Comment by stweil** [comment]
Related: #3095.

Citing the documentation on `/D`: "Must be the last parameter on the command line and must not contain quotes even if the path contains blank spaces." In your case it was the last parameter, but you used quotes!

**Comment by qzd1989** [comment]
> Related: #3095.
> 
> Citing the [documentation](https://nsis.sourceforge.io/Which_command_line_parameters_can_be_used_to_configure_installers%3F) on `/D`: "Must be the last parameter on the command line and must not contain quotes even if the path contains blank spaces." In your case it was the last parameter, but you used quotes!

Sir, I tried, but it still did not work.
<img width="1600" alt="image" src="https://github.com/user-attachments/assets/2a391c62-3519-4270-abef-61c879ebd2f5">

**Comment by stweil** [comment]
Right, this looks like an issue in the NSIS configuration. Maybe it should not set `MULTIUSER_INSTALLMODE_INSTDIR` because this setting overrides the command line value.

**Comment by qzd1989** [comment]
> Right, this looks like an issue in the NSIS configuration. Maybe it should not set `MULTIUSER_INSTALLMODE_INSTDIR` because this setting overrides the command line value.

Thank you, Sir, it seems there are no answers for now. I'll continue to pay attention to this issue.
